packages/headless/src/features/attached-results/attached-results-state.ts

```diff
@@ -48,6 +48,7 @@ export interface AttachedResultsState {
   results: AttachedResult[];
   /**
    * A loading state, used to sync loading the attached results between multiple components.
+   * To be removed in UI-KIT V4 TODO: SFINT-6404
```

Comment:
That's the usual syntax of a TODO comment:
```ts
// TODO: KIT-3391 - export other SSR commerce controllers
```

This is important because your code editor will recognize this and highlight in a special way different from a normal comment so it's more visible. with this syntax your todo becomes readable by some tools that will enumerate the todo list in a given repo, you can explore this by playing with the VS code extension: https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree